tolgee-cli/faq.mdx

```diff
@@ -0,0 +1,101 @@
+---
+id: faq
+title: Frequently Asked Questions
+---
+
+This page addresses common questions and misconceptions about the Tolgee CLI.
+
+## Key Management and Default Values
+
+### Q: Does the CLI detect duplicate keys with different default values?
+
+**A: No, currently the CLI does not detect this scenario.** 
+
+If you have the same key with different default values in your code:
+
+```jsx
+<T keyName="save-btn" defaultValue="Save" />
+<T keyName="save-btn" defaultValue="Save (Changed)" />
+```
+
+The CLI will not warn you about this conflict. This is a known limitation.
+
+**Recommended approach**: Use consistent default values across your codebase, or better yet, rely on the Tolgee Platform as the single source of truth for all translations.
+
+### Q: Why don't default values automatically update my base language translations?
+
+**A: This is by design to prevent conflicts between code and platform translations.**
+
+When you first sync a key, the default value becomes the base language translation. However, subsequent changes to default values in code will not update the platform translation. This prevents situations where:
+
+- A developer changes a default value in code
+- A translator or PM has already refined the wording in the platform
+- The two sources of truth conflict with each other
+
+### Q: Can I search for strings in my code if I only store keys?
+
+**A: This depends on your approach to translation management.**
+
+If you follow Tolgee's recommended approach of using the platform as the single source of truth:
+- You would search for keys in your code: `grep -r "save-btn" src/`
```

Comment:
Yoi don't need to search for the key usually, because you can click it directly in the app. So the correct approach is to click the string in the app and then search for the key.

Reply:
Since this is tle CLI docs, you should mention that in-context is only for the native JS SDKs.